.docker/qgis_resources/test_runner/qgis_setup.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+# Setup QGIS for the automated tests
+# This is normally called from Travis or rundockertests.sh
+# before running the tests for a particular plugin
+#
+# Note: on QGIS3 assumes the default profile for root user
+#
+# - create the folders
+# - install startup.py monkey patches
+# - disable tips
+# - enable the plugin
+
+PLUGIN_NAME=$1
+CONF_FOLDER="/root/.config/QGIS"
+CONF_FILE="${CONF_FOLDER}/QGIS2.conf"
```

Comment:
QGIS 2?

Reply:
I guess we can drop it now,  the script was originally supporting both 2 and 3.